src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenOnBehalfOfParameters.cs

```diff
@@ -10,6 +10,7 @@ internal class AcquireTokenOnBehalfOfParameters : IAcquireTokenParameters
     {
         public UserAssertion UserAssertion { get; set; }
         public bool SendX5C { get; set;}
+        public bool ForceRefresh { get; set; }
```

Comment:
Please add a line in LogParamters about this (and maybe also log the UserAssertion.Hash)

Reply:
done. 
not sure why to log the hash...is there a reason for that? i have not included it.

---
In reply to: [532528452](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2250#discussion_r532528452) [](ancestors = 532528452)